src/sidebar/store/modules/selection.js

```diff
@@ -162,6 +168,19 @@ const update = {
     };
   },
 
+  CHANGE_FOCUS_MODE_USER: function(state, action) {
+    return {
```

Comment:
Does this need any commenting?

Reply:
I think its a strong no, because one can trace the action type back to the action and read that doc. We don't have many comments in reducers in general even in `annotations.js`